runtime/vm/ValueTypeHelpers.hpp

```diff
@@ -456,6 +456,8 @@ class VM_ValueTypeHelpers {
 
 	/**
 	 * Performs an aaload operation on an object. Handles flattened and non-flattened cases.
+	 * This function could call into J9AllocateObject() which might trigger GC (in the slow path), so receiverObject might be moved by GC.
```

Comment:
This only occurs if `fast` is false

Reply:
Yes, I have `(in the slow path)` later in this sentence.